### PR TITLE
message id generation: hash only the event and not the uuid

### PR DESCRIPTION
**Pull request by andrewloyola**
- [x] I've included a changeset (psst. run `yarn changeset`. Read about changesets [here](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md)).

### CODE REVIEW

File: packages/browser/src/core/events/index.ts [hunk 1]
```diff
@@ -261,7 +261,7 @@ export class EventFactory {
       context,
       integrations: allIntegrations,
       ...overrides,
-      messageId: 'ajs-next-' + md5.hash(JSON.stringify(event) + uuid()),
+      messageId: 'ajs-next-' + md5.hash(JSON.stringify(event)) + uuid(),
```
Comment (silesky) [inline packages/browser/src/core/events/index.ts:264]: Maybe we just rip the band-aid off and do what we do in node.js?

https://github.com/segmentio/analytics-next/blob/master/packages/node/src/lib/get-message-id.ts

Comment (andrewloyola) [reply packages/browser/src/core/events/index.ts]: updated the pr to reflect this change

**Comment by silesky** [comment]
@andrewloyola thanks for your contribution!
Couple things:
1. Can we yarn uninstall the md5 dependency?
2. Can you run "yarn test" and fix any broken tests?

**Comment by silesky** [comment]
Closing in favor of #1045

**Pull request closed without merging**